conf/base.config

```diff
@@ -44,6 +44,8 @@ process {
     cpus = { check_max( 10 * task.attempt, 'cpus' ) }
     memory = { check_max( 80.GB * task.attempt, 'memory' ) }
     time = { check_max( 8.h * task.attempt, 'time' ) }
+    errorStrategy = { task.exitStatus in [1,143,137,104] ? 'retry' : 'terminate' }
+    maxRetries = 2
```

Comment:
I'm surprised that the default for all processes is only 1. I think it would be good to set that to 2, then again we don't need this here.